stake-o-matic/src/main.rs

```diff
@@ -709,6 +856,116 @@ fn process_confirmations(
     ok
 }
 
+const DATA_CENTER_ID_UNKNOWN: &str = "0-Unknown";
```

Comment:
If a bunch of nodes end up in unknown, they won't get destaked right?

Reply:
They would ATM. I can make them not, but didn't initially because I was unsure how easy it is to manipulate the data and make a cozy spot to hide

Reply:
Are there lots of unknowns right now?  If not then it probably doesn't matter

Reply:
I think there were like five

Reply:
Added logging for >3%